lib/cluster/corosync.c

```diff
@@ -688,6 +688,18 @@ corosync_cluster_name(void)
     return cluster_name;
 }
 
+/*!
+ * \internal
+ * \brief Check whether Corosync has any configuration keys with given prefix
+ *
+ * \param[in] prefix  Configuration key prefix
+ *
+ * \return -1 on error, 0 if no such key present, 1 if any such key present
+ * \note This function always returns the result of the first time it was
+ *       called, even if later called with a different section name (which
+ *       obviously should never be done), and even if the corosync configuration
+ *       has since been reloaded.
```

Comment:
Longer term, this should be refactored to address the issues in the note, but I wanted to minimize the changes for 2.0.5

Reply:
Maybe toss a FIXME on here, unless you've got it on some sort of list.

Reply:
I do hope to get to it reasonably soon, just didn't want to go overboard with changes in 2.0.5